### PR TITLE
DDD-20-UI Order form without validation

### PR DESCRIPTION
**Pull request by Accir**
Created order form without validation on user input

### CODE REVIEW

File: ui/src/components/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    setCurrentPage(currentPage + 1);
+    setProgress(progress + 25);
+  }
+
+  function PreviousPage(e) {
+    setCurrentPage(currentPage - 1);
+    setProgress(progress - 25);
+  }
+
+  function displayError(errorMessage, type) {
```
Comment (Zygis0321) [inline ui/src/components/Form.jsx]: To make error messages reusable in other components I would consider moving this function somewhere else. Maybe some wrapper component could work?

**Review by Zygis0321 — COMMENTED** [review]
Great job! The only thing I would improve is making error messages reusable

**Review by giplonis — APPROVED** [review]

File: ui/src/components/Form.jsx (hunk 1 above)
Comment (Accir) [reply ui/src/components/Form.jsx]: I can't move it to other component or seperate function as it requires a hook or a prop from parent function to call a snackbar, only solution is to pass the function to children components if you want to call it or use context to call it globally. If you have any better ideas, feel free to commit. [Library documentation](https://iamhosseindhv.com/notistack) if you are interested.

**Comment by Accir** [comment]
Nicely done 👍 , if everything else is ok, merge the PR

**Pull request merged**